llama_parse/base.py

```diff
@@ -254,6 +262,12 @@ async def _create_job(self, file_path: str, extra_info: Optional[dict] = None) -
         job_id = response.json()["id"]
         return job_id
 
+    @staticmethod
+    def __get_filename(f: TextIOWrapper | AbstractBufferedFile) -> str:
```

Comment:
Any reason for this to be a static method? Or have a double underscore name? (also, should use union typing here again)